src/main/java/me/botsko/prism/database/sql/SqlSelectIDQueryBuilder.java

```diff
@@ -14,13 +14,13 @@
  * Created for use for the Add5tar MC Minecraft server
  * Created by benjamincharlton on 31/03/2019.
  */
-public class SQLSelectIDQueryBuilder extends SQLSelectQueryBuilder implements SelectIDQuery {
+public class SqlSelectIDQueryBuilder extends SqlSelectQueryBuilder implements SelectIDQuery {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Abbreviation in name 'SqlSelectIDQueryBuilder' must contain no more than '2' consecutive capital letters.](https://app.codacy.com/gh/AddstarMC/Prism-Bukkit/pullRequest?prid=5438808)